README.md

```diff
@@ -275,9 +275,12 @@ TODO
 
 ### Fake MESH
 
-Clone https://github.com/mattd-kainos/fake-mesh from our fork
+Documentation and Source Code: https://github.com/mattd-kainos/fake-mesh
```

Comment:
Shouldn't we use some kind of NHS/Kainos repo for storing it?

Reply:
yes it should go into nhsd github eventually. This work for now though.